### PR TITLE
Add link to optunahub-registry

### PR DESCRIPTION
**Pull request by not522**
## Motivation

We released the beta version of OptunaHub and would like to encourage contributions to that as well.

## Description of the changes

This PR will add a sentence to `CONTRIBUTING.md` encouraging contributions to `optunahub-registry`.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Join Us!
 If you feel like giving a hand, here are some ways:
 - Implement a feature
     - If you have some cool idea, please open an issue first to discuss design to make your idea in a better shape.
+    - We also welcome PRs for [optunahub-registry](https://github.com/optuna/optunahub-registry), which is a service for sharing user-defined Optuna packages.
```
Comment (y0z) [inline CONTRIBUTING.md:19]: I feel optunahub-registry itself is not a service.

```suggestion
    - We also welcome PRs for [optunahub-registry](https://github.com/optuna/optunahub-registry). [OptunaHub](https://hub.optuna.org/) is a feature-sharing platform for Optuna.
```

**Review by y0z — COMMENTED** [review]
I leave a comment.

**Comment by not522** [comment]
Thank you. I applied your suggestion. PTAL.

**Review by y0z — APPROVED** [review]
LGTM

**Review by HideakiImamura — APPROVED** [review]

**Pull request merged**